docs/concepts/artifacts.md

```diff
@@ -0,0 +1,227 @@
+
+### Create Table Artifacts
+
+You can create a table artifact by calling `create_table_artifact()`.
+
+```python
+from prefect import flow, task
+from prefect.experimental.artifacts import create_table_artifact
+
+def my_fn():
+    highest_churn_possibility = {
+        'customer_id': ['123456789', '987654321', '246810121', '135791113', '864208046'],
+        'name': ['John Smith', 'Jane Doe', 'Bob Johnson', 'Sarah Jones', 'Tom Wilson'],
+        'churn_probability': [0.85, 0.79, 0.67, 0.61, 0.57]
+    }
+
+    create_table_artifact(
+        key="personalized-reachout",
+        table=highest_churn_possibility,
+        description= "# Marvin, please reach out to these customers today!"
+    )
+
+if __name__ == "__main__":
+    my_fn()
+```
+
+As you may notice, you don't need to create an artifact in a flow run context. You can use it however you like and still get the benefits in the Prefect UI.
+
+## Managing Artifacts
+
+### Reading Artifacts
+
+In the Prefect UI, you can view all of the latest versions of your artifacts and click into a specific artifact to see its lineage over time. You can also use the [Prefect REST API](https://app.prefect.cloud/api/docs#tag/Artifacts/operation/read_artifacts_api_accounts__account_id__workspaces__workspace_id__experimental_artifacts_filter_post) to programmatically filter your results.
+
+### Deleting Artifacts
+
+To delete an artifact, simply do so directly in the Artifacts page in the UI. Alternatively, you can delete artifacts using the [Prefect REST API](https://app.prefect.cloud/api/docs#tag/Artifacts/operation/delete_artifact_api_accounts__account_id__workspaces__workspace_id__experimental_artifacts__id__delete).
+
+## Artifacts API
+
+Prefect provides the [Prefect REST API](https://app.prefect.cloud/api/docs#tag/Artifacts) to allow you to create, read, and delete artifacts programmatically. With the Artifacts API, you can automate the creation and management of artifacts as part of your data pipeline.
+
+To use it, remember to enable artifacts by running:
+```bash
+prefect config set PREFECT_EXPERIMENTAL_ENABLE_ARTIFACTS=True
+```
+
+For example, to read the 5 most recently created markdown, table, and link artifacts, you can do the following:
+
+```python
+import requests
+
+PREFECT_EXPERIMENTAL_ENABLE_ARTIFACTS=True
+PREFECT_API_URL="https://api.prefect.cloud/api/accounts/abc/workspaces/xyz"
+PREFECT_API_KEY="pnu_ghijk"
+data = {
+    "sort": "CREATED_DESC",
+    "limit": 5,
+    "artifacts": {
+        "key": {
+            "exists_": True
+        }
+    }
+}
+
+headers = {"Authorization": f"Bearer {PREFECT_API_KEY}"}
+endpoint = f"{PREFECT_API_URL}/experimental/artifacts/filter"
+
+response = requests.post(endpoint, headers=headers, json=data)
+assert response.status_code == 200
+for artifact in response.json():
+    print(artifact)
+```
+
+If we don't specify a key or that a key must exist, we will also return results (which are a type of key-less artifact).
+
```

Comment:
@znicholasbrown mentioned that we shouldn't mention or elaborate on results being artifacts. Should we leave the above sentence as is, then? Or just remove it?

Reply:
Ah then that answers my earlier question of whether we want to link the two or not. If that's the case I'd suggest removing "making it easy to see where and when your results were written." from line 3 of  [docs/concepts/artifacts.md](https://github.com/PrefectHQ/prefect/pull/8980/files/189aa4f0731710308b6b62a1944e19cbd3207c05#diff-2afc6f000359fc8fc7a7b28f7595439c86a12ed14587e09edec93c140e9d1f07) Even if you're not specifically referencing results as the concept of results, it may be confusing.

Reply:
Thanks, I have update the first paragraph. I will this sentence here for now then.